src/Dependencies/PooledObjects/Microsoft.CodeAnalysis.PooledObjects.shproj

```diff
@@ -1,7 +1,7 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <Project ToolsVersion="14.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
   <PropertyGroup Label="Globals">
-    <ProjectGuid>cd77a8cc-2885-47a7-b99c-fbf13353400b</ProjectGuid>
+    <ProjectGuid>c1930979-c824-496b-a630-70f5369a636f</ProjectGuid>
```

Comment:
I've seen it in a few PRs.  Someone probably copy pasted a project, and VS is generating a new Guid for it because there is a clash with something else in the solution.